DirectEmployers/MyJobs
# [PD-3659] control access to new site traffic with AppAccess

## What's this PR do?
Allows certain users to access the new site-traffic reporting feature through a new AppAccess called "Site Traffic V2"

## Where should the reviewer start?
`myjobs/models.py` L#1495

## How should this be manually tested?

1. In shell run: `from myjobs.models import update_activities; update_activities()`
2. Go to Django admin and add `Site Traffic V2` to your company.
3. Navigate to https://secure.my.jobs/reports/view/overview/ 
You should see *4* intentions under Analytics, the bottom two are for site-traffic, old and new, both should work.
4. Go to Django admin, revoke `Site Traffic V2` from your company, and go back to reports overview, you should only see 3 intentions under Analytics.

## Any background context you want to provide?
- This change will allow some members to preview the new site-traffic reporting, during which phase they will have access to both new and old versions. Once we decide to roll out to everyone, just update the AppAccess, clean up a bit, which is trivial. 
- Requires running `update_activities()` to work
